### PR TITLE
updated referral email text

### PR DESCRIPTION
**Pull request by wsvincent**

### CODE REVIEW

File: layouts/thanks/single.html [hunk 1]
```diff
@@ -154,8 +154,8 @@ <h1 class="linespot" hidden>Thank you! You are #<span id="SignupNumber">1000</sp
...
-        var body = encodeURIComponent('I just signed up for this awesome service, Meta, and knew you’d love it!\n\nDid you know the average employee wastes 4 hours each week searching for and recreating lost work?\n\nMeta allows you to access any files in seconds, wherever it is. It connects with your Google Drive, Dropbox, OneDrive, and Evernote accounts so you can search for any of your files on these services- all from Meta.\n\nPlus, Meta makes search way more intuitive. No more having to remember where you saved a document or what you named it.\n\nReady to stop wasting 4 hours each week? Check it out! Use this link to sign up for Meta and we both win!\n\n' + link);
+        var subject = encodeURIComponent('A Friend invited You to Meta!');
+        var body = encodeURIComponent('Hi there,\n\nI’ve been using Meta and thought you might like it. Meta gives you one place to search for all your files, whether they’re on your computer or in the cloud! It connects seamlessly with your Google Drive, Dropbox, Gmail, Slack, Evernote, and Trello accounts.\n\nSign up with my link to get early access to the beta:\n\n' + link);
```
Comment (jasonbriggs) [inline layouts/thanks/single.html]: At this point they won't really have been using Meta yet, right? Otherwise this looks good!

Comment (wsvincent) [inline layouts/thanks/single.html]: Good point. I'll make a tweak...

**Comment by kujenga** [comment]
Rebased this to the tip of master to bring it up to date

**Comment by kujenga** [comment]
👍

**Pull request merged**